include/zephyr/bluetooth/hci.h

```diff
@@ -97,6 +97,17 @@ int bt_hci_get_conn_handle(const struct bt_conn *conn, uint16_t *conn_handle);
  */
 int bt_hci_get_adv_handle(const struct bt_le_ext_adv *adv, uint8_t *adv_handle);
 
+/** @brief Obtain the version string given a core version number.
+ *
+ * The core version of a controller can be obtained by issuing
+ * the HCI Read Local Version Information command.
+ *
+ * @param core_version The core version.
+ *
+ * @return Version string corresponding to the core version number.
+ */
+const char *bt_hci_get_ver_str(uint8_t core_version);
```

Comment:
Did you consider making an `enum` for the available versions? 

It's by no means obvious that `4` means "2.1"

Reply:
I didn't consider that, but it really makes sense. 
I see that the convention in `hci_types.h` is to rather have defines than enums though.

Reply:
Ah, right, sorry, didn't see we were in `hci.h` - Yes, here it is common _not_ to use `enum`s (at least for now. 

In which case, we can just refer to the existing `#define`s for versions